containers/heading/UpgradeButton.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { c } from 'ttag';
+
+import { Icon } from 'react-components';
+
+const UpgradeButton = ({ className }) => {
+    return (
+        <a to="/settings/subscription" className={className}>
+            <Icon name="upgrade-to-paid" className="topnav-icon mr0-5 flex-item-centered-vert fill-white" />
```

Comment:
Since the `Icon` is formatted to be in the top bar, I'm wondering if it makes sense to keep `className` as a prop.

Reply:
I'll update it next time I work on header navigation. 👍